### ISSUE TITLE
Getting pixels from taggedImage

### ISSUE THREAD

**Issue by nanthony21** [micro-manager/pycro-manager#22]
the `pix` field of a `TaggedImage` appears to just be a copy of the `tags` metadata.

```
>>> a = bridge.get_core().get_tagged_image()
>>> a.pix == a.tags
        True
>>> print(a.pix)
{'Z-Description': 'Demo stage driver',
 'Camera-PixelType': '16bit',
 'Camera-Binning': '1',
 'Core-Shutter': 'Shutter',
 'Camera-FastImage': '0',
 'D-DA-HubID': '',
...
```

**Comment by henrypinkard** [comment]
I wasn't getting this behavior, but I did get something else strange where calling
```
a.pix
```
yielded
```
Out[2]: <property at 0x7fcae8eed5e8>
```
instead of a numpy array. This seems to have been caused by https://github.com/micro-manager/pycro-manager/pull/16, so I reverted to the old code for now. https://github.com/micro-manager/pycro-manager/pull/23

**Comment by nanthony21** [comment]
fixed in #19
